components/portal/main/partials/header.html

```diff
@@ -19,36 +19,47 @@
 
 -->
 <!-- PRIORITY NOTIFICATIONS -->
-<notifications-bell slot="myuw-notifications" mode="priority" class="priority-gt-xs" ng-if="!guestMode" hide-xs></notifications-bell>
-
-<myuw-app-bar
-    theme-name="{{ portal.theme.title }}"
-    app-name="{{ appBarTitle }}"
-    app-url="{{ MISC_URLS.rootURL }}"
-    ng-class="{ 'has-priority-notifications' : hasPriorityNotifications }">
-
-    <!-- MAIN MENU TOGGLE -->
-    <md-button slot="myuw-navigation"
-        class="menu-toggle"
-        ng-controller="MenuToggleController as vm"
-        ng-click="vm.showMainMenu()"
-        ng-class="{ 'menu-margin' : vm.isMenuConfigured }"
-        ng-show="vm.isMenuConfigured"
-        ng-cloak>
-        <md-icon>menu</md-icon>
-        <md-tooltip md-direction="bottom" md-delay="500" md-autohide class="top-bar-tooltip">Menu</md-tooltip>
-        <notifications-bell mode="mobile-bell" ng-if="!guestMode" hide-gt-xs></notifications-bell>
-    </md-button>
-
-    <!-- SEARCH BUTTON (XS-ONLY) -->
-    <search ng-show="APP_FLAGS.showSearch"  slot="myuw-search" mode="mobile-search" layout="row" layout-align="end center" hide-gt-xs></search>
-
-    <!-- SEARCH BAR (SM+) -->
-    <div slot="myuw-search" class="header__search-bar-container" md-colors="{ backgroundColor: '{{portal.theme.name}}-primary' }" layout="row" flex-gt-xs="100" layout-align="center center" hide-xs>
-      <search ng-show="APP_FLAGS.showSearch"></search>
+<notifications-bell slot="myuw-notifications" mode="priority" class="priority-gt-xs" ng-if="!guestMode" hide-xs>
+</notifications-bell>
+
+<myuw-app-bar theme-name="{{ portal.theme.title }}" app-name="{{ appBarTitle }}" app-url="{{ MISC_URLS.rootURL }}"
+  ng-class="{ 'has-priority-notifications' : hasPriorityNotifications }">
+
+  <!-- MAIN MENU TOGGLE -->
+  <md-button slot="myuw-navigation" class="menu-toggle" ng-controller="MenuToggleController as vm"
+    ng-click="vm.showMainMenu()" ng-class="{ 'menu-margin' : vm.isMenuConfigured }" ng-show="vm.isMenuConfigured"
+    ng-cloak>
+    <md-icon>menu</md-icon>
+    <md-tooltip md-direction="bottom" md-delay="500" md-autohide class="top-bar-tooltip">Menu</md-tooltip>
+    <notifications-bell mode="mobile-bell" ng-if="!guestMode" hide-gt-xs></notifications-bell>
+  </md-button>
+
+  <!-- SEARCH BUTTON (XS-ONLY) -->
+  <search ng-show="APP_FLAGS.showSearch" slot="myuw-search" mode="mobile-search" layout="row" layout-align="end center"
+    hide-gt-xs></search>
+
+  <!-- SEARCH BAR (SM+) -->
+  <div slot="myuw-search" class="header__search-bar-container"
+    md-colors="{ backgroundColor: '{{portal.theme.name}}-primary' }" layout="row" flex-gt-xs="100"
+    layout-align="center center" hide-xs>
+    <search ng-show="APP_FLAGS.showSearch"></search>
+  </div>
+
+  <myuw-help myuw-help-title="Help links" slot="myuw-help" show-button>
+    <div slot="myuw-help-content">
+      <ul>
+        <li>
+          <a href="https://outages.doit.wisc.edu/">Check the Outages page</a>
```

Comment:
The most important thing to say is this is fine demo content and we should just try it.

As we tighten this up towards something to cut a release of and ship, will need to figure out how to make this help content differ in Madison vs System contexts or include links appropriate to both. Cf. the different help links and experiences you get accessing <https://my.wisc.edu> vs <https://my.wisconsin.edu>

Reply:
I added more meaningful links, found in the footer of MyUW. Would it be helpful to also include the link to https://kb.wisconsin.edu/myuwsystem/page.php?id=15303 or https://kb.wisconsin.edu/myuwsystem/ ? As we have it in the footer of https://my.wisconsin.edu.

Reply:
Further updated links in ff9c6b3 .